pkg/backup/backup.go

```diff
@@ -62,6 +62,9 @@ type Backupper interface {
 	// Backup takes a backup using the specification in the velerov1api.Backup and writes backup and log data
 	// to the given writers.
 	Backup(logger logrus.FieldLogger, backup *Request, backupFile io.Writer, actions []velero.BackupItemAction, volumeSnapshotterGetter VolumeSnapshotterGetter) error
+	BackupWithResolvers(log logrus.FieldLogger, backupRequest *Request, backupFile io.Writer,
+		backupItemActions framework.BackupItemActionResolver, itemSnapshotters framework.ItemSnapshotterResolver,
```

Comment:
The name of the resolver parms are inconsistent with `RestoreWithResolvers` you should probably rename `backupItemActions` to `backupItemActionResolvers` and `itemSnapshotters` to `itemSnapshotterResolvers`

Reply:
Sure